sitestats/sitestats-tool/src/java/org/sakaiproject/sitestats/tool/wicket/components/ImageWithLink.java

```diff
@@ -42,15 +43,23 @@ public ImageWithLink(String id, String imgUrl, String lnkUrl, String lnkLabel, S
 		boolean exists = (lnkTarget != null && lnkLabel != null && lnkUrl != null);
 		ExternalLink lnk = null;
 		if(exists) {
-			add( new ExternalImage("image", imgUrl).setVisible(imgUrl != null) );
+			if(imgUrl != null) {
```

Comment:
We shoudln't need `imgUrl` now the null check neither. Also I feel like the component is confusing now. Should we rename it? And was it alway used just with the file icon?

Reply:
yeah pretty confusing. maybe we just get rid of it!